electricitymap/contrib/config/model.py

```diff
@@ -340,3 +342,54 @@ def _load_config_model() -> ConfigModel:
 CO2EQ_CONFIG_MODEL = CO2eqConfigModel(
     direct=CO2EQ_PARAMETERS_DIRECT, lifecycle=CO2EQ_PARAMETERS_LIFECYCLE
 )
+
+
+class DataCenter(StrictBaseModel):
+    displayName: str
+    lonlat: tuple[float, float] | None
+    provider: str
+    region: str
+    status: str
+    zoneKey: ZoneKey
```

Comment:
Id include an array of aliases here, GCP data centers for example are split up in A, B, C sections which could be nice to capture if we are going to make a lookup API for them.